chainerx_cc/chainerx/cuda/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ CUDA_ADD_LIBRARY(chainerx_cuda STATIC
     cuda_device/dot.cu
     cuda_device/exp_log.cu
     cuda_device/fill.cu
+    cuda_device/hyperbolic.cu
```

Comment:
There might be no need to split files with `trigonometric.cu`. How about merging them?

Reply:
Oh will merge them with `trignometric`